homeassistant/components/apple_tv/media_player.py

```diff
@@ -36,14 +36,18 @@
 
 from . import AppleTVEntity, AppleTVManager
 from .browse_media import build_app_list
-from .const import DOMAIN
+from .const import DOMAIN, KNOWN_PLAYERS
 
 _LOGGER = logging.getLogger(__name__)
 
 PARALLEL_UPDATES = 0
 
 # We always consider these to be supported
-SUPPORT_BASE = MediaPlayerEntityFeature.TURN_ON | MediaPlayerEntityFeature.TURN_OFF
+SUPPORT_BASE = (
+    MediaPlayerEntityFeature.TURN_ON
+    | MediaPlayerEntityFeature.TURN_OFF
+    | MediaPlayerEntityFeature.GROUPING
```

Comment:
I don't think this should be added here as it would suggest all devices supports grouping, which isn't the really the case (old Apple TVs for instance).

You need to add the features related to grouping to SUPPORT_APPLE_TV. You can just map all of them to GROUPING and the issue above will be solved.

Reply:
I'll explain my reasoning:

Apple devices can be controllers, that actually play the media and send it to other devices. Other AirPlay devices can be players.

Home Assistant has a few features gated behind this support flag, one of which is the `group_members` attribute.
A device that can be grouped needs to have this flag set so that it can expose that attribute.

In `async_join_players` there is an extra check for `FeatureName.AddOutputDevices` to see if the device can be the group controller, that feature is set If there is an MRP connection (so not for AirPlay only devices).

Now I don't have any device that doesn't work with grouping so I can't run any more checks.
@postlund do you know if feature support is device specific, or OS version specific? Also can these old Apple TVs be added to a group over AirPlay even if they can't be leaders? Are there any AirPlay devices that cannot be added t a group? If I knew that I could try to add more specific features flagging.